pkg/model/postgresql_deployment.go

```diff
@@ -226,7 +226,7 @@ func PostgresqlDeploymentReconciled(cr *v1alpha1.Keycloak, currentState *v13.Dep
 			VolumeMounts: []v1.VolumeMount{
 				{
 					Name:      PostgresqlPersistentVolumeName,
-					MountPath: "/var/lib/postgresql/data",
+					MountPath: "/var/lib/pgsql/data",
```

Comment:
Let's make sure this error won't happen again - could you please put this in `constants.go` and refactor the other place where this occurs?